app/models/image.rb

```diff
@@ -0,0 +1,4 @@
+class Image < ActiveRecord::Base
+  belongs_to :image_set
+  belongs_to :image_type
+end
```

Comment:
@iezer I'll definitely port those over. I didn't realize I was duplicating efforts there - sorry about that.  I'll take a look through the s3-upload-rebased branch and bring over the validation. I think I'll leave the unit tests on your branch as I don't have the Fabricator infrastructure, but I'm happy to update your branch with this schema if that'd be helpful.